### PR TITLE
Fix arrival at isochron

### PR DESCRIPTION
**Pull request by djludo**

### CODE REVIEW

File: source/routing/raptor_api.cpp [hunk 1]
```diff
@@ -750,7 +751,8 @@ make_nm_response(RAPTOR &raptor, const std::vector<type::EntryPoint> &origins,
 
                 std::vector<type::StopPoint*> stop_points;
                 for(auto& path : paths) {
-                    const type::StopPoint* sp = path.items[0].stop_points[0];
+                    const type::StopPoint* sp = clockwise ? path.items.front().stop_points.front() :
+                                                             path.items.back().stop_points.back();
                     stop_points.push_back(const_cast<type::StopPoint*>(sp));
```
Comment (TeXitoi) [inline source/routing/raptor_api.cpp:756]: Please remove this `const_cast`, change `stop_points` to `std::vector<const type::StopPoint*>` and `add_isochrone_response` using `std::vector<const type::StopPoint*>`.

**Comment by buildbotctp** [comment]
Can one of the admins verify this patch?

File: source/routing/raptor_api.cpp (hunk 1 above)
Comment (djludo) [inline source/routing/raptor_api.cpp:756]: the "make_isochron" method calls "add_isochron" with raptor.data.pt_data->stop_points as parameter, that's the reason of the const_cast

**Comment by TeXitoi** [comment]
Jenkins, test this please

**Pull request merged**